src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20251112113245_Routine_PauseResumableIndexRebuilds.sql

```diff
@@ -0,0 +1,53 @@
+CREATE OR ALTER PROCEDURE dbo.PauseResumableIndexRebuilds
+    @Tables nvarchar(MAX) -- This is a comma-delimited list of table names e.g. "Observation, ObservationFilterItem".
+AS
+BEGIN
+    SET NOCOUNT ON;
```

Comment:
I think it would also be worth adding `SET NOCOUNT ON` to the `RebuildIndexes` procedure.